eosdac-material/README.md

```diff
@@ -30,8 +36,22 @@ npm install
 ```
 cp src/statics/config.jungle.json src/statics/config.json
 ```
+
 **6. Run dev server with material theme**
 ```
 quasar dev
 ```
 [Quasar Docs](https://quasar-framework.org/guide/index.html)
+
+## What to Change
+Since eosDAC is making this open source software for the use of creating other DACs, DACs will need to know what to change in order to quickly edit the software for their needs. The main repositories you will want to change for your DAC's branding, messaging, and assets are:
+- `src/statics`
+- `src/i18n`
+- `src/statics/config.json`
+
+`statics` is the main repository where you will want to change branding assets. Replace the files and retain the file names so you do not have to go through the other areas of the code to update the file names. You will also need to update the `i18n` directory for proper messaging and translations for the DAC's written content. The `config.json` file aligns with the proper deployed smart contract and you will need to change this file and the templates towards your DAC's smart contract.
+
+Other odd ball files may need to be changed by your team.
```

Comment:
Thanks for the latest changes.

Maybe I'm not comfortable with "odd ball files" being in a professional README? Can we change this to be more specific about updating view files if they want to change the structure or layout? That could include the layouts, pages, plugins, etc. I'm thinking it may be better to leave this out entirely or make it specific instead of something vague. That's my preference anyway.

While in here, let's go ahead and remove the npm references since it just seems to be causing problems.

Reply:
What npm reference?

Reply:
In the readme. No worries, I'll take care of it.